rclcpp/include/rclcpp/subscription.hpp

```diff
@@ -99,6 +103,26 @@ class SubscriptionBase
   virtual const std::shared_ptr<rcl_subscription_t>
   get_intra_process_subscription_handle() const;
 
+  // RCLCPP_PUBLIC
+  template<typename EventCallbackT>
+  void
+  add_event_handle(
```

Comment:
Same question as above. Are we adding a handle to an event or a handler for events?

Reply:
Will rename to `add_event_handler()`.